sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.F

```diff
@@ -4421,51 +4319,31 @@ SUBROUTINE SPFFT1(IMAX,INCW,INCG,KMAX,W,G,IDIR)
 !> Read input global 30-arc second orography data.
 !!
 !! @param[out] glob The orography data.
-!! @param[in] itopo Not used.
 !! @author Jordan Alpert NOAA/EMC
-      subroutine read_g(glob,ITOPO)
+      subroutine read_g(glob)
       implicit none
-cc
-      integer*2    glob(360*120,180*120)
-cc
-      integer      ix,jx
-      integer      ia,ja
-cc
-      parameter   (ix=40*120,jx=50*120)
-      parameter   (ia=60*120,ja=30*120)
-cc
-      integer*2    idat(ix,jx)
-      integer      itopo
-cc
-      integer      i,j,inttyp
-cc
-      real(kind=8) dloin,dlain,rlon,rlat
-cc
-      open(235, file="./fort.235", access='direct', recl=43200*21600*2)
-      read(235,rec=1)glob
-      close(235)
-cc
+
+      include 'netcdf.inc'
+
+      integer*2, intent(out) :: glob(360*120,180*120)
+
+      integer :: ncid, error, id_var, fsize
+
+      fsize=65536
+
+      error=NF__OPEN("fort.235",NF_NOWRITE,fsize,ncid)
```

Comment:
same as above, read the actual file name?

Reply:
Done at 2ae071c.